### DISCUSSION TITLE
Vcf to 23andme, ftdna, ancestry converter

### DISCUSSION THREAD

**Discussion by georggf** [WGSExtract/WGSExtract.github.io#15] [Ideas]
Hi! 
Thanks very much for your great project. 
It was very useful for converting bam to 23andme, ftdna, ancestry format. 
But recently I've got original vcf file from non wgs provider and couldn't convert it correctly with some third-party application in 23andme format for ftdna and myheritage upload. 
Would you please consider creating extra pipeline in your great wgse because I think you would better handle such work

**Comment by RandyHarr** [comment]
Generally, you cannot get a usable microarray / genotyping file from a VCF.  It will not pass various vendors filters looking for heterozygous call minimums.  When creating from a VCF, you have to fill in for the values desired in the microarray that do not exist in the VCF. These could be due to the location not being tested or it is homozygous to the reference.  So you either have to put in '--' (no call) or fill in a homozygous value set from the reference.  Either solution is usually rejected by the sites due to carefully tuned counters they have for the number of no-calls, homozygous, heterozygous, and homozygous to the reference. This filters were added when security researchers saw that you could put in a file that was heterozygous at all locations. Which would then match everyone in the database.

Free DNA Kit Studio (a MS Windows program from DNAGenics) has a VCF to microarray format tool.  You are welcome to start experimenting with that. You can let us know your success in generating an accepted file in the Facebook group or here.
